### ISSUE TITLE
i3ipc-glib-1.0 

### ISSUE THREAD

**Issue by Foggalong** [denesb/xfce4-i3-workspaces-plugin#18]
The autogen script is saying that the `i3ipc-glib` package isn't installed even though it is. It's also the latest version from git (0.6) so I'm not sure why it's requesting 1.0

I'm not really sure how to fix this - I tried running `sudo env PKG_CONFIG_PATH=/usr/local/src/i3ipc-glib ./configure` but that just provided the same error. Any help would be much appreciated.

```
checking for i3ipc-glib-1.0 >= 0.5... not found
*** The required package i3ipc-glib-1.0 was not found on your system.
*** Please install i3ipc-glib-1.0 (atleast version 0.5) or adjust
*** the PKG_CONFIG_PATH environment variable if you
*** installed the package in a nonstandard prefix so that
*** pkg-config is able to find it.
```

**Comment by denesb** [comment]
The 1.0 version is probably just a typo from my part, it is looking for version >= 0.5.
I think the reason you have problems is because you installed i3ipc-glib in /usr/local which is a non-standard path. I had this problem too and I solved it by installing i3ipc-glib to /usr.

**Comment by Foggalong** [comment]
That was indeed the problem - installing in /usr worked. Now I have the problem of the indicator being installed but not actually working properly. Should I open a new issue or carry on here?

**Comment by denesb** [comment]
What is the problem exactly? You can describe it here and if it's something new I'll open a new issue for it.

**Comment by Foggalong** [comment]
The plugin works fine on launch, starting with a 1 for workspace 1 and then as you switch up to a higher workspace it moved up with it. But then as soon as you move to a lower workspace it freezes on whatever workspace you got to :(

Is also doesn't have an option to change the text colour from black to white which would be useful for dark pannels.

**Comment by denesb** [comment]
Which version of the plugin are you using? The one from git, or one of the releases?
[1.1.1](https://github.com/denesb/xfce4-i3-workspaces-plugin/releases/tag/1.1.1) should work fine, the latest version from the master branch has regressions as I'm rewriting some of the code.
Also, you can change the text color and set different text colors for the normal, focused and urgent states. Go to Panel Preferences -> Items, Select the workspace plugin and then click preferences.
Unfortunately accessing the plugin config via right-clicking the buttons on the panel is very hard, because if you click on the buttons they consume the click event.

**Comment by Foggalong** [comment]
I was using the git version. My bad! The preferences wasn't working but I'm guessing again that's because I was using it from git. I'll download 1.1.1 and then report back whether I'm still having the problem :)

**Comment by Foggalong** [comment]
Got 1.1.1 installed now and everything is working perfectly. I thought getting XFCE panel and i3 workspaces working together would be such a specific use case that I'd have to do it myself, so much thanks for making this and releasing it!

**Comment by denesb** [comment]
Yeah, there are all kinds of people and they use weird combinations of software and even write plugins to make them work together smoother... :). By the way I just released 1.2.0 , it should work even better then 1.1.1.

**Comment by Foggalong** [comment]
OOo thanks! I'll give it a spin tomorrow :)
